src/protocol/primitives.rs

```diff
@@ -478,6 +478,61 @@ where
     }
 }
 
+#[derive(Debug)]
+pub struct Bytes(pub Vec<u8>);
```

Comment:
Looks good. Please copy over the docs from <https://kafka.apache.org/protocol#protocol_types> and add a roundtrip test.